controllers/complianceeventsapi/migrations/000001_compliance_history_initial_tables.up.sql

```diff
@@ -44,7 +45,7 @@ CREATE TABLE IF NOT EXISTS compliance_events(
    compliance TEXT NOT NULL,
    message TEXT NOT NULL,
    timestamp TIMESTAMP NOT NULL,
-   metadata TEXT,
+   metadata JSON,
```

Comment:
Good call. I originally left it open to TEXT so that we could support SQLite too but we seem to be leaning towards just supporting Postgres.

Reply:
@JustinKuli I think JSONB might be better here for efficient querying.